.idea/MonteCinema.iml

```diff
@@ -47,6 +47,7 @@
     <orderEntry type="library" scope="PROVIDED" name="crass (v1.0.6, RVM: ruby-3.0.1) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="diff-lcs (v1.4.4, RVM: ruby-3.0.1) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="erubi (v1.10.0, RVM: ruby-3.0.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="factory_bot (v6.2.0, RVM: ruby-3.0.1) [gem]" level="application" />
```

Comment:
IMHO you shouldn't keep your personal editor settings in the repository.

It's a good candidate for local or global `.gitignore` entry. I encourage you to read [this](https://dart.dev/guides/libraries/private-files#details)

Reply:
+1 👍